requirements-lock.txt

```diff
@@ -0,0 +1,44 @@
+#
```

Comment:
If don't think there is much need for this file either, there is no reason to freeze the package version at this point.

Reply:
The purpose of this file is to create reproductibility. People don't need to use exactly the same versions present in the lock file, but in the case there is something wrong or behaving badly, it becomes way easier to identify the source of code breaks. Imagine a situation where the update of a dependency breaks the package. That is identified first by Travis CI tests failing, then a dev can compare failing requiremets lock and last passing one and rollback one dependency at a time to identify what broke it.

Reply:
Okay this makes sense, can we make it as a hidden file ?

Reply:
Sure. It was set as hidden in the latest pull request draft.